sdv/constraints/base.py

```diff
@@ -161,6 +161,8 @@ def fit(self, table_data):
             table_data (pandas.DataFrame):
                 Table data.
         """
+        self._fit(table_data)
```

Comment:
In `_fit` the `_constraint_columns` might be changed. Id the part below runs before this, it will try to access the wrong `_constraint_columns`. Before there were no constraints that changed `_constraint_columns` in `_fit` so this wasn't a problem